_define.php

```diff
@@ -45,4 +45,8 @@
     null                        //Permissions needed - not yet implemented
 );
 
+// override global variables
+global $adhesion_form_url;
```

Comment:
I'd prefer to avoid `global` usage since I'm trying to drop them all (see http://bugs.galette.eu/issues/414)